src/modules/Dropdown/Dropdown.js

```diff
@@ -296,6 +296,10 @@ export default class Dropdown extends Component {
     /** The text displayed in the dropdown, usually for the active item. */
     text: PropTypes.string,
 
+    openOnFocus: PropTypes.bool,
```

Comment:
Let's add some descriptions to these:

"Whether or not the menu should open when the dropdown is focused."

And:

"Whether or not the menu should close when the dropdown is blurred."